assets/css/components/_news-and-views.scss

```diff
@@ -0,0 +1,37 @@
+figure {
+  height: rem(298px);
+}
+
+/* resize images */
+.news-item-img {
+  display: block;
+  margin: 0 auto;
+  max-height: 100%;
+  max-width: 100%;
+}
+
+.news-item {
+  padding: rem(8) rem(34);
+}
+
+.news-grid {
```

Comment:
With this change, the parent-children relationship is clearer. Thanks.

Could you apply the same change to all elements in this scss file? The example is:

```
.news-grid {
  // styles for .news-grid

  .api-content {
     // styles for .api-content

    .news-item {
      // styles for .news-item

      figure {
        // for figure
          .news-item-img {
          }
        }
      }
    }
  }
}
```

The same comment applies to the desktop styles.